lib/capistrano-db-tasks/database.rb

```diff
@@ -106,8 +106,13 @@ def dump_cmd_ignore_data_tables_opts
   class Remote < Base
     def initialize(cap_instance)
       super(cap_instance)
-      @config = @cap.capture("cat #{@cap.current_path}/config/database.yml")
-      @config = YAML.load(ERB.new(@config).result)[@cap.fetch(:rails_env).to_s]
+      @cap.within @cap.current_path do
+        @cap.with rails_env: @cap.fetch(:rails_env) do
+          @config = @cap.capture(:rails, 'runner "puts Rails.application.config.database_configuration[Rails.env].to_yaml"', '2>/dev/null')
+          @config.gsub!(/.*\-\-\-/m, '---') # Remove all bundler and rails initialization errors
+        end
```

Comment:
Maybe I am missing something.. But is there a reason to use rails runner here? Rather than just depending on the :environment task or something? This feels hacky.. perhaps what @fabn was feeling as well..